### PR TITLE
Add bedtools

### PR DESCRIPTION
**Pull request by StephDC**
Great tool that could get me into my bed earlier.

### CODE REVIEW

File: extra-scientific/bedtools/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,4 @@
+PKGNAME=bedtools
+PKGSEC=science
+PKGDES="A powerful tool for genome arithmetic"
+export prefix=/usr
```
Comment (MingcongBai) [inline extra-scientific/bedtools/autobuild/defines]: Use `ABMK="prefix=/usr"`, but in fact this is already defined internally. That said, this line could be removed.

File: extra-scientific/bedtools/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,4 @@
+PKGNAME=bedtools
+PKGSEC=science
+PKGDES="A powerful tool for genome arithmetic"
```
Comment (MingcongBai) [inline extra-scientific/bedtools/autobuild/defines]: Where did the dependencies go?

File: extra-scientific/bedtools/autobuild/defines (hunk 1 above)
Comment (StephDC) [inline extra-scientific/bedtools/autobuild/defines]: Tested on ACBS that the prefix was not defined. Maybe due to the differences in capitalization? From the makefile, it takes prefix but not PREFIX.

File: extra-scientific/bedtools/autobuild/defines (hunk 2 above)
Comment (StephDC) [inline extra-scientific/bedtools/autobuild/defines]: Seems like that only the make test requires samtools as described in Gentoo's ebuild. It just makes on my instance so I don't know if there would be any other dependencies...

Comment (MingcongBai) [inline extra-scientific/bedtools/autobuild/defines]: > Seems like that only the make test requires samtools as described in Gentoo's ebuild. It just makes on my instance so I don't know if there would be any other dependencies...

Well to say the least, if it's a compiled ELF binary, that would require `glibc` (which we will not neglect) - just run `ldd` and see for yourself.

File: extra-scientific/bedtools/autobuild/defines (hunk 1 above)
Comment (MingcongBai) [inline extra-scientific/bedtools/autobuild/defines]: > Tested on ACBS that the prefix was not defined. Maybe due to the differences in capitalization? From the makefile, it takes prefix but not PREFIX.

Then, this is still not the way to do it. Add a line `MAKE_AFTER="prefix=/usr"`.

**Comment by MingcongBai** [comment]
The tree is quite a mess... While the PR is fine. Next time please try and `git pull --rebase` when upstream branch (in this case `staging`) is updated.

**Comment by MingcongBai** [comment]
The tree is quite a mess... While the PR is fine. Next time please try and `git pull --rebase` when upstream branch (in this case `staging`) is updated.

**Pull request merged**